spec/models/member_spec.rb

```diff
@@ -1,8 +1,48 @@
 require 'spec_helper'
 
 describe Member do
+  subject { Fabricate(:member) }
+
+  it { should belong_to(:user) }
+  it { should belong_to(:organization) }
+  it { should have_one(:account) }
+  it { should delegate_method(:balance).to(:account).with_prefix }
+  it { should delegate_method(:gender).to(:user).with_prefix }
+  it { should delegate_method(:date_of_birth).to(:user).with_prefix }
+
+  it { should validate_presence_of(:organization_id) }
+  it { should validate_presence_of(:member_uid) }
+
+  describe "#offers" do
+    let(:member) { Fabricate(:member) }
+    let(:member_offer) { Fabricate(:offer,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.